pkgs/applications/graphics/goxel/default.nix

```
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
     description = "Open Source 3D voxel editor";
     homepage = https://guillaumechereau.github.io/goxel/;
     license = licenses.gpl3;
-    platforms = [ "x86_64-linux" ]; # see https://github.com/guillaumechereau/goxel/issues/125
+    platforms = [ platforms.linux ];
```

It's not working because `stdenv.lib.platforms.linux` is a list.

```suggestion
    platforms = platforms.linux;
```